ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoData.java

```diff
@@ -40,20 +41,22 @@
     private RaoDataManager raoDataManager;
     private ReferenceProgram referenceProgram;
     private GlskProvider glskProvider;
+    private Set<Country> loopflowCountries;
```

Comment:
is this redundant with RaoParameters' field or do we absolutely need this here too for loopflow computation ?

Reply:
a lot of methods have access to raoData but not the parameters.
It's now used in the Leaf's constructor to get the parent's list of loopflow countries